app/src/main/res/values/strings.xml

```diff
@@ -50,6 +50,10 @@
     <string name="crop_screen_crop_button">Validate</string>
     <string name="crop_screen_retry_button">Retry</string>
 
+    <!-- GlanceScreen -->
+    <string name="glance_title">Glance</string>
+    <string name="glance_description">Glance actions.</string>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/LunabeeStudio/Lunabee_Compose_Android/blob/6c09cd48d00137585ef42b09da4d94f399394e06/app/src/main/res/values/strings.xml#L55"></span>The resource <code>R.string.glance_description</code> appears to be unused</td>
    </tr>
  </tbody>
</table>

Reply:
Il a raison